FeatureSuggestions/FirozAhmad.md

```diff
@@ -0,0 +1,14 @@
+## Feature Suggestions
+
+
+1. Actors: There will be four actors Admin,Head of Department(HOD),Teacher and Students in our system.
+
+2. Login: Every user will have to Login.After successful Login,each user will go to their respective webpage.
+
+3. Role of Admin: After successful login of Admin, he can add new student detail,new head of department,new teacher detail.After successful creation of student ,head of department, and teacher ,a new username and password wiil be created by sytem and will be send to that user(student/HOD/teacher) and it will also be saved to database.He can view the list of HOD under his institute, teachers under repective HODs,and students under respectives teacher in his courses taken by teacher.
+
+4.Role of HOD:After successful login,he can update attendance of teachers of his department.
+
+4.Role of Teacher: After successful login of Teacher, he will update attendance of his student of his respective courses taken by him.He can also update his profile detail like password.
```

Comment:
There is a difference in attendance at school and college. At school, attendance is taken at the class level, and college attendance is taken at the course level. We should address that. Also, the same system should work wherein the office manager can trace the attendance of the employees.